.releaserc.js

```diff
@@ -64,6 +64,21 @@ const releaseReplaceSetting = [
         ],
         countMatches: true,
       },
+      {
+        files: ['dist/js/inpage.js'],
+        from: 'CORE_EXTENSION_VERSION',
+        // Replace CORE_EXTENSION_VERSION string to the next release number in the inpage.js file
+        to: `<%= _.replace(nextRelease.version, /[^0-9.]/g, '') %>`,
+        results: [
+          {
+            file: 'dist/js/inpage.js',
+            hasChanged: true,
+            numMatches: 1,
+            numReplacements: 1,
+          },
+        ],
+        countMatches: true,
```

Comment:
We want to let Core Web know which extension version they're interacting with.
This is so Core Web can send compatible requests (some requests, like `bitcoin_sendTransaction`, have changed the payload shape).

Here is a related PR:
* https://github.com/ava-labs/core-web-properties/pull/3114